src/components/sbb-datepicker-toggle/sbb-datepicker-toggle.e2e.ts

```diff
@@ -36,6 +36,33 @@ describe('sbb-datepicker-toggle', () => {
     expect(tooltip).toEqualAttribute('data-state', 'opened');
   });
 
+  it('renders and opens tooltip programmatically', async () => {
+    const page: E2EPage = await newE2EPage({
+      html: `
+          <sbb-datepicker-toggle date-picker="datepicker" disable-animation></sbb-datepicker-toggle>
+          <sbb-datepicker input='datepicker-input' id='datepicker' value="01-01-2023"></sbb-datepicker>
+          <input id="datepicker-input">
+        `,
+    });
+    const element: E2EElement = await page.find('sbb-datepicker-toggle');
+    const tooltipTrigger: E2EElement = await page.find(
+      'sbb-datepicker-toggle >>> sbb-tooltip-trigger'
+    );
+    const tooltip: E2EElement = await page.find('sbb-datepicker-toggle >>> sbb-tooltip');
+    await page.waitForChanges();
+    expect(element).toHaveClass('hydrated');
+    expect(tooltipTrigger).not.toHaveAttribute('disabled');
+    expect(tooltip).toEqualAttribute('data-state', 'closed');
+
+    await page.evaluate(() =>
+      (document.querySelector('sbb-datepicker-toggle') as HTMLSbbDatepickerToggleElement).open()
+    );
+
+    await page.waitForChanges();
+
+    expect(tooltip).toEqualAttribute('data-state', 'opened');
```

Comment:
shouldn't we wait for didopen event fired to make the test more stable?

Reply:
On it, should I implement this also in other tests where we check if the tooltip is open?

Reply:
hmm maybe, but apparently they did not fail :D

Reply:
This one seems to be okay too, but I'll add this check anyway since it makes sense